ruby/vampires.rb

```diff
@@ -0,0 +1,74 @@
+puts "How many employees will be processed?"
+employees = gets.chomp.to_i
+
+employees.times do |x|
+  puts "What is your name?"
+  username = gets.chomp 
+  
+  puts "How old are you?"
+  age = gets.chomp.to_i
+  
+  puts "What year were you born?"
+  birth_year = gets.chomp.to_i
+  
+  actual_age = Time.now.year - birth_year
+  correct_age = actual_age == age
+  
+  puts "Our company cafeteria serves garlic bread. Should we order some for you?"
+  garlic = gets.chomp
+  
```

Comment:
The above with correct age is entirely correct and really well done.

For garlic and insurance we are missing the boolean values as these are just strings.

`garlic = gets.chomp`
`likes_garlic = garlic == 'Y'`